rust/chains/abacus-ethereum/src/validator_manager.rs

```diff
@@ -105,8 +114,13 @@ where
                 .iter()
                 .map(|s| s.to_vec().into())
                 .collect(),
+            message.to_vec().into(),
+            sol_proof,
+            proof.index.into(),
         );
-
-        Ok(report_tx(tx).await?.into())
+        let gas = tx.estimate_gas().await?.saturating_add(U256::from(100000));
```

Comment:
100,000?

Reply:
yeah this is super weird. It's a relic of the old process logic so I'm inclined to leave as is for now